src/main.rs

```diff
@@ -103,20 +107,32 @@ impl FileId {
     }
 }
 
+#[derive(PartialEq)]
+enum WriteMode {
+    AskConfirm,
+    #[allow(dead_code)]
+    Force,
+}
+enum SenderEvents {
```

Comment:
```suggestion
enum SenderEvent {
```
Every instance contains only a single event.